### PR TITLE
Updated chess android Readme.md 

### LINKED ISSUE #89: Update README.md file for Chess android app
Readme.md file for chess android app needs to be updated with the latest changes. Need to have clear steps on how to build and run the app from scratch.

### PR DESCRIPTION
**Pull request by Vishwa4jeet**
Fixes #89 

Made changes to the readme file as per the latest code changes.

### CODE REVIEW

File: chess/README.md [hunk 1]
```diff
@@ -1,39 +1,81 @@
...
+Alternatively,
+
+2. **From the command prompt**:
 ```shell
-$ cd DCAP-Sapphire-Examples/chess/
-> cat >> local.properties  << EOF
-ndk.dir=<your ndk dir>
-sdk.dir=<your sdk dir>
-EOF
+    $ cd DCAP-Sapphire-Examples/hanksTodo/
```
Comment (VenuReddyHuawei) [inline chess/README.md]: change this to chess in next PR.

**Comment by VenuReddyHuawei** [comment]
LGTM

File: chess/README.md (hunk 1 above)
Comment (Vishwa4jeet) [reply chess/README.md]: Done.

File: chess/README.md [hunk 2]
```diff
@@ -1,39 +1,81 @@
...
+    > cat >> local.properties  << EOF
```
[the remark this answers is not in the document]
Comment (Vishwa4jeet) [reply chess/README.md]: Done.

**Pull request merged**